modin/utils.py

```diff
@@ -47,6 +47,58 @@
 """
 
 
+Invocable = list
```

Comment:
this feels super-weird, but I guess it's because Dask is weird...

Reply:
Yes, unfortunately making this `Invocable = tuple` didn't work (I think because of the depth-first evaluation behavior I mentioned above).

Reply:
Sure, it might make sense to move it. Did you have a better place in mind for this?